README.md

```diff
@@ -17,29 +18,33 @@ Visit the [wiki](https://github.com/inkfarer/ipl-overlay-controls/wiki) if you a
 1. Install NodeCG: `nodecg setup`
 
 1. Install ipl-overlay-controls: `nodecg install inkfarer/ipl-overlay-controls`.
-	
-	* To specify a version, append `#<tag>` after the repository name.
 
-		Example: `nodecg install inkfarer/ipl-overlay-controls#1.3.2`
+  * To specify a version, append `#<tag>` after the repository name.
 
-		Find a list of the repository's releases [here](https://github.com/inkfarer/ipl-overlay-controls/releases).
```

Comment:
I'm unsure what happened to the formatting here: https://github.com/inkfarer/ipl-overlay-controls/blob/e3caabb0cca5789660559752ed5fe1508fc364fd/README.md